howdoi/howdoi.py

```diff
@@ -335,12 +325,15 @@ def _get_questions(links):
     return [link for link in links if _is_question(link)]
 
 
-def _get_answer(args, link):
-    cache_key = _get_cache_key(link)
+def _get_answer(args, link):  # pylint: disable=too-many-branches
+    cache_key = link
     page = cache.get(link)  # pylint: disable=assignment-from-none
     if not page:
+        print(f'Fetching page: {link}')
```

Comment:
We only want to display this when explainability is enabled.

Reply:
Remember: explainability statements will be logged with `logging.info` because we don't want this information to be printed unless the `--explain` flag is set.